Remotion/Web/ClientScript/Scripts/DropDownMenu.ts

```diff
@@ -105,38 +105,51 @@ class DropDownMenu
 
   public static AddMenuInfo(menuInfo: DropDownMenu_MenuInfo): void
   {
+    ArgumentUtility.CheckNotNullAndTypeIsObject ('menuInfo', menuInfo);
+
     DropDownMenu._menuInfos[menuInfo.ID] = menuInfo;
   }
 
-  public static BindOpenEvent (openTarget: JQuery, menuID: string, eventType: string, getSelectionCount: Nullable<DropDownMenu_SelectionCountGetter>, moveToMousePosition: boolean): void
+  public static BindOpenEvent (openTargetOrSelector: CssSelectorOrElement<HTMLElement>, menuID: string, eventType: string, getSelectionCount: Nullable<DropDownMenu_SelectionCountGetter>, moveToMousePosition: boolean): void
   {
-    ArgumentUtility.CheckNotNullAndTypeIsJQuery ('openTarget', openTarget);
+    ArgumentUtility.CheckNotNull ('openTargetOrSelector', openTargetOrSelector);
     ArgumentUtility.CheckNotNullAndTypeIsString ('menuID', menuID);
     ArgumentUtility.CheckNotNullAndTypeIsString ('eventType', eventType);
+    ArgumentUtility.CheckTypeIsFunction ('getSelectionCount', getSelectionCount);
     ArgumentUtility.CheckNotNullAndTypeIsBoolean ('moveToMousePosition', moveToMousePosition);
 
-    var menuButton = openTarget[0].tagName.toLowerCase() === 'a' ? openTarget : openTarget.find ('a[href]:last');
-    menuButton.attr('aria-haspopup', 'menu');
-    menuButton.attr('aria-expanded', 'false');
+    const openTarget = ElementResolverUtility.ResolveSingle (openTargetOrSelector);
+
+    var menuButton: HTMLAnchorElement;
+
+    if (openTarget.tagName.toLowerCase() === 'a')
+    {
+      menuButton = openTarget as HTMLAnchorElement;
+    }
+    else
+    {
+      menuButton = Array.from(openTarget.querySelectorAll<HTMLAnchorElement>('a[href]')).slice(-1)[0];
+    }
```

Comment:
I meant expanding to an `if-else` in order to write a more understandable .last()

```suggestion
    if (openTarget.tagName.toLowerCase() === 'a')
    {
      menuButton = openTarget as HTMLAnchorElement;
    }
    else
    {
      const menuButtons = openTarget.querySelectorAll<HTMLAnchorElement> ('a[href]');
      menuButton = menuButtons[menuButtons.length - 1];
    }
```